### ISSUE TITLE
Tab order after locking database is broken

### ISSUE THREAD

**Issue by keylogthis** [keepassxreboot/keepassxc#12190]
### Have you searched for an existing issue?

- [x] Yes, I tried searching and reviewed the pinned issues

### Brief Summary

After locking a database the cursor focus goes into the password input box and it appears that the tab focus is on "I have a key file" but it is not because pressing enter attempts to unlock instead of opening the key file open file, and the only way to get the tab focus back onto "I have a key file" is to wrap all the way around all the controls back to it. It's extremely annoying and frustrating for me who doesn't want to use my mouse, and wants to keep my hands on the keyboard to unlock the database again. There should be a consistent tab order... and ideally for me it would be the first tab focuses keyfile.

This only happens after locking an unlocked database. If you are initially opening a database the tab order is okay, and you can focus keyfile with two tab presses. 

### Steps to Reproduce

Unlock a database
Lock it

### Expected Versus Actual Behavior

I expect to be able to select key file with one or two tab presses. Not have to cycle through the entire set of controls to get to it.

### KeePassXC Debug Information

```Text

```

### Operating System

Linux

### Linux Desktop Environment

XFCE

### Linux Windowing System

X11

**Comment by droidmonkey** [comment]
Pressing enter does not activate a link, pressing Space should. Enter is usurped by the overall widget to unlock the database.

**Comment by droidmonkey** [comment]
Although after re reading this might be a weird quirk

**Comment by keylogthis** [comment]
Yeah, it seems if I press tab twice it does go back and I can hit enter to get the open file dialog for the keyfile, that's good enough for me.
